### ISSUE TITLE
docx documents get suddenly corrupted

### ISSUE THREAD

**Issue by hoegerl** [ONLYOFFICE/DocumentServer#1185]
**Do you want to request a *feature* or report a *bug*?**
-> bug report

**What is the current behavior?**
Some .docx files get suddenly corrupted. We're using NC20.0.8, onlyoffice/documentserver 6.2.0.123, ONLYOFFice-app 6.3.0.
The corrupted file also shows corrupted on another NC/onlyoffice instance.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
The file is beeing worked on by multiple users. The rendering of the file ends with an annotation comment. There is no repro for the bug. We could work on a docx document without problem. From one day to the other the file got corrupted.

We managed to get the file working again by opening it in MS Word, saving it as .doc. Then reopeneing the .doc file and save as .docx. From now on this file shows correct again in NC/onlyoffice editor.

**What is the expected behavior?**
docx documents don't get corrupted.

**Did this work in previous versions of DocumentServer?**
We had this error also in 5.x version.

**DocumentServer version:**
6.2.0.123

**Operating System:**
Unbuntu 18.04 LTS

**Browser version:**
Multiple, latest versions of Safari, Firefox, Chrome

I've edited the raw XML to anonymize the text-content of the attached file.
[corrupted.docx](https://github.com/ONLYOFFICE/DocumentServer/files/6163687/corrupted.docx)

**Comment by ShockwaveNN** [comment]
You've attached corrupted.docx file - is this file is the result of corruption or original document which was correupted after some actions?

Could you attach screenshot of error about corruption?
And take a look at logs at /var/log/onlyoffice/ for any error message with timestamp of corruption happened

**Comment by hoegerl** [comment]
I did some more research on the behaviour since the circumstances & actions what lead to the corruption are unclear.
I have a 7MB large docx which is finde opening in MS Word locally. I upload this to nextcloud. I can download the document from nextcloud, it will be 7MB and working in MS Word (oo-corrupted_7mb_ms-word.png).

I then open it in the oo-editor via Browser, and it only displays parts of the first page (oo-currupted_nextcloud_browser-editor.png). After this i download the file via Nextcloud and it still is 7MB and looks ok in MS Word.

Now i open the document in the oo-editor again, and choose file -> download as -> docx and the file is only 37KB!

![oo-corrupted_7mb_ms-word](https://user-images.githubusercontent.com/64417258/112328732-e5aa1e80-8cb6-11eb-8e90-747a0a2c1e0d.png)
![oo-currupted_nextcloud_browser-editor](https://user-images.githubusercontent.com/64417258/112328737-e642b500-8cb6-11eb-8333-44ab1981e69b.png)

**Comment by ShockwaveNN** [comment]
@hoegerl So judging by size of file - you've attached courreped version

Could you attach original one?
Maybe remove sensitive information, replace it with some other info, but error shall persist for file

**Comment by hoegerl** [comment]
Unfortunately i can not attach the original document since the whole document is confidential, In the meantime i try to shorten the document and see if the error still occurs. This way i would be able to anonymize the short document and upload it.

**Comment by ShockwaveNN** [comment]
@hoegerl Ok we'll waiting for some document from you, because without file I don't think to be able to catch your bug

**Comment by ShockwaveNN** [comment]
This issue was closed due to no response.

If you'll able to add some file - I'll reopen this issue

**Comment by adsche** [comment]
Dear @ShockwaveNN , we have experienced the same problem multiple times with different Powerpoint presentations.

I've attached a minimal example that opens fine in Powerpoint, opens fine in ONLYOFFICE when uploaded to Nextcloud, but is defect when downloaded again via "File > Download As > PPTX". It still works fine if downloaded directly from Nextcloud.

This is with Docker image docker.io/onlyoffice/documentserver:latest, Nextcloud 29.0.7 and the ONLYOFFICE Nextcloud App version 9.4.0 (but I suspect this could also be triggered without Nextcoud at all).

[Reduced-6.pptx](https://github.com/user-attachments/files/17220462/Reduced-6.pptx)

Defect file downloaded from ONLYOFFICE: 
[Reduced-6-dl.pptx](https://github.com/user-attachments/files/17220468/Reduced-6-dl.pptx)


My initial assessment as a layperson: The slide contains an animation referencing (as far as I can tell) an object that isn't there anymore. So it is somewhat gone bad, but this was not "crafted" this way, users somehow end up in this situation by normal Powerpoint use and, more importantly, the file still works fine in Powerpoint.
